exec/java-exec/src/main/java/org/apache/calcite/jdbc/DynamicRootSchema.java

```diff
@@ -46,12 +47,14 @@
 public class DynamicRootSchema extends DynamicSchema {
   private static final Logger logger = LoggerFactory.getLogger(DynamicRootSchema.class);
 
+  private static final String ROOT_SCHEMA_NAME = "";
```

Comment:
@vvysotskyi 
Is there a way to get this PR to do what you want without having to maintain a modified version of Calcite?  My concern is that as Drill progresses, it will be much easier to maintain if we don't have to maintain a custom version of Calcite.

Reply:
This is not a Calcite class. It is just Drill class, placed in the Calcite package to be able to access some classes and fields. It was introduced in fix for DRILL-5089.

Reply:
Ah ok.  In that case, LGTM +1